resources/cluster-essentials/charts/pod-preset/Chart.yaml

```diff
@@ -1,4 +1,6 @@
 name: pod-preset
-description: A Helm chart for PodPreset resource that's implemented via a CRD that is paired with a mutating webhook in order to modify pod specs on the fly
+description: A Helm chart for PodPreset resource that's implemented via a CRD that
```

Comment:
For a multiline string shouldn't this look like

```yaml
description: |
  foo
```

Reply:
thx it looks like autoformatter